components/resolved_ts/src/advance.rs

```diff
@@ -127,6 +133,7 @@ impl<E: KvEngine> AdvanceTsWorker<E> {
     async fn region_resolved_ts_store(
```

Comment:
Could you update the related code in cdc or add a TODO comment there?

Reply:
Done